theano/tensor/signal/conv.py

```diff
@@ -88,11 +75,9 @@ def conv2d(input, filters, image_shape=None, filter_shape=None,
 
     new_filter_shape = tensor.join(0, tensor.stack([sym_nkern, 1]), filters.shape[-2:])
     filters4D = tensor.reshape(filters, new_filter_shape, ndim=4)
-
-    # perform actual convolution ###
-    op = conv.ConvOp(output_mode=border_mode,
-                     dx=subsample[0], dy=subsample[1],
-                     imshp=imshp, kshp=kshp, nkern=nkern, bsize=bsize, **kargs)
+    op = AbstractConv2d(imshp=imshp, kshp=kshp,
```

Comment:
Here imshp and kshp don't have the same definition. It must be a 4 element list, not a 2 or 3 element list. Can you convert them?

thanks.